ccd-definition/AuthorisationCaseField/CareSupervision/barrister.json

```diff
@@ -852,7 +852,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "courtBundleTabHeading",
+    "CaseFieldID": "hearingDocumentsTabHeading",
```

Comment:
Think we might have to be careful renaming here - might need to deprecate the courtBundleTabHeading

Reply:
check with Somesh and Daniel. normally, we don’t need to move the fields to deprecated if no data migration is required.
But better move them to DEPRECATED if the field could technically still be hanging around on some case data.